src/snoowrap.js

```diff
@@ -769,6 +769,20 @@ const snoowrap = class snoowrap {
   getNewComments (subredditName, options) {
     return this._getSortedFrontpage('comments', subredditName, options);
   }
+  
+  
+  /**
+   *  @summary Get list of content by IDs.
+   *  @param {string[]} ids An array of fullname IDs you want to fetch the content of.
+   *  @returns {Promise} A listing of content requested, can be any class fetchable by API. e.g. Comment, Submission
+  */
+  getContentByIDs (ids) {
```

Comment:
Could we make it possible to use Submission and Comment objects too? We would need to change the JSDoc param to `@param {Array<string|Submission|Comment>} ids` and make sure that fullname is fetched from the objects if there are any